.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -15,25 +15,41 @@ about: Did something not work?
 - [ ] I have searched the issue tracker for an issue that matches the one I want
       to file, without success.
 
-## Description of the problem
+## Description of the issue
 
-## How has this issue affected you? What are you trying to accomplish?
+<!--
+Please provide an overview of the problem
+-->
+
+## How has this issue affected you?
+
+<!--
+What were you trying to do when you encountered this issue?
+-->
 
-### To Reproduce
+## Steps to reproduce the issue
 
 <!--
-Your best chance of getting this bug looked at quickly is to provide an example.
+To help us look into this bug quickly, please provide a list of steps we
+can follow to reproduce the issue.
+
+For example:
+
+1. On Safari, right click on the editor
+2. Click "Copy"
+3. The value copied to your clipboard does not match the selected text
 -->
 
-### Link to sandbox: [link]() (optional)
```

Comment:
This heading was inconsistent with other sections, so I removed it as a heading and put it inside the "Steps to reproduce the issue" section